DataConnectors/Digital shadows/DigitalShadowsConnectorAzureFunction/DS_poller.py

```diff
@@ -41,10 +48,21 @@ def get_data(self):
         """
         triage_id = []
         try:
-            event_dataJSON = self.DS_obj.get_triage_events(str(self.before_time), str(self.after_time))
-            event_data = json.loads(event_dataJSON)
+            if(isinstance(self.event, int)):
+                event_dataJSON = self.DS_obj.get_triage_events_by_num(self.event)
+                event_data = json.loads(event_dataJSON)
+                self.date.post_event(self.event + int(len(event_data[:20])))
+            else:
+                event_dataJSON = self.DS_obj.get_triage_events(str(self.before_time), str(self.after_time))
+                event_data = json.loads(event_dataJSON)
+                event_num = int(event_data[0]['event-num'])
```

Comment:
Do we need to convert it to int?

Reply:
Yes because in next step I am incrementing its value.